src/main/kotlin/no/risc/github/GithubHelper.kt

```diff
@@ -314,4 +325,22 @@ class GithubHelper(
         riScId: String,
         branch: String,
     ): String = "/$owner/$repository/commits?sha=$branch&path=$riScFolderPath/$riScId.$filenamePostfix.yaml"
+
+    fun uriToFetchCommits(
+        owner: String,
+        repository: String,
+        branch: String? = null,
+        path: String? = null,
+    ): String =
+        if (branch == null && path == null) {
+            "/$owner/$repository/commits"
+        } else {
+            "/$owner/$repository/commits?${ branch?.let { "sha=$branch" } ?: ""}&${ path?.let { "path=$path" } ?: "" }"
+        }
+
+    fun uriToFetchCommitsSinceLastPublishedRiScDateTime(
+        owner: String,
+        repository: String,
+        lastPublishedRiScDateTime: OffsetDateTime,
+    ): String = "/$owner/$repository/commits?since=$lastPublishedRiScDateTime"
```

Comment:
Her tror jeg vi burde bruke et mer generisk navn, da denne kan brukes utenom bare for `LastPublishedRiScDateTime`:

```suggestion
    fun uriToFetchCommitsSince(
        owner: String,
        repository: String,
        since: OffsetDateTime,
    ): String = "/$owner/$repository/commits?since=$since"
```

Reply:
ok